client/components/EntryFormRow/index.js

```diff
@@ -1,20 +1,57 @@
 import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { getCategories } from 'modules/categories';
+import { actions as AppActions } from 'modules/transactions';
 import DataSelector from './DataSelector';
 import styles from './style.scss';
 
+@connect(
```

Comment:
I think we should keep the components as presentation-only. Can we move the state connection and handlers to a new EntryFormRow container instead? Then we can either use the new Container/EntryFormRow.js right in the BudgetGrid component or do some composing and pass it in as a child from the BudgetContainer.  @grgur might have different thoughts...